src/app/components/ContactViewProperty.js

```diff
@@ -57,8 +57,10 @@ const ContactViewProperty = ({
             );
         }
         if (field === 'url') {
+            // use new root address when the url does not include the protocol (HTTP or HTTPS)
+            const href = value.startsWith('http') ? value : `//${value}`;
```

Comment:
Not sure how it will behave if the user enter an URL starting with `//` like `//google.com`. But it should be rare...